### PR TITLE
feat: add panel component

### PR DESCRIPTION
**Pull request by IliaSolovev**

### CODE REVIEW

File: src/components/navigation-panel/navigation-panel.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import tw, { styled } from 'twin.macro';
+
+export const NavigationPanel = () => {
+  console.log('asd');
```
Comment (RomanAchilov) [inline src/components/navigation-panel/navigation-panel.tsx:5]: ?

File: src/components/navigation-panel/navigation-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  return <Wrapper />;
+};
+
+const Wrapper = styled.div`
```
Comment (IliaSolovev) [inline src/components/navigation-panel/navigation-panel.tsx:9]: delete file

File: src/components/panel/panel.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+export const Panel = ({
+  children, header, isOpen, onClosePanel, footer,
+}: Props) => (isOpen
+  ? (
+    <>
+      <div tw="absolute top-0 left-12 z-20 text-monochrome-light-tint text-24 h-full left-12 flex flex-col">
+        <div tw="px-6 py-7 leading-32 bg-monochrome-black">{header}</div>
+        <div tw="bg-monochrome-black flex-grow" style={{ opacity: '0.97' }}>{children}</div>
```
Comment (RomanAchilov) [inline src/components/panel/panel.tsx:34]: use jit

File: src/components/panel/panel.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        {footer && <div tw="h-18 bg-monochrome-black">{footer}</div>}
+      </div>
+      <div onClick={onClosePanel} tw="absolute z-10 inset-0 left-12" style={{ background: 'rgba(0, 0, 0, 0.4)' }} />
```
Comment (RomanAchilov) [inline src/components/panel/panel.tsx:37]: use jit

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

**Review by RomanAchilov — APPROVED** [review]

**Pull request merged**